api/routes/discovery.py

```diff
@@ -153,6 +153,9 @@ def stages(self):
         stages.append(
             {'$set': {'score': {'$meta': 'searchScore'}, 'highlights': {'$meta': 'searchHighlights'}}},
         )
+        if self.term:
+            # get only results with relevance score > 1.5 (which seems to be a good threshold)
+            stages.append({'$match': {'score': {'$gt': 1.5}}})
```

Comment:
@pkdash - As discussed on Slack today, 1.4 is probably a better starting value. We'll also need to add an environment variable for this parameter, and the ability to change it with minimal disruption.

Reply:
@Maurier Using env variable now for the threshold.